src/graph/network.h

```diff
@@ -21,9 +21,9 @@ using dgl::runtime::NDArray;
 namespace dgl {
 namespace network {
 
-// Max size of message queue for communicator is 200 MB
+// Max size of message queue for communicator is 500 MB
 // TODO(chao): Make this number configurable
-const int64_t kQueueSize = 200 * 1024 * 1024;
+const int64_t kQueueSize = 500 * 1024 * 1024;
```

Comment:
this also needs to be fixed. we shouldn't set a hard-coded maximal size.

Reply:
I fix this by giving user an argument in python.